src/expandableCalendar/WeekCalendar/presenter.js

```diff
@@ -0,0 +1,187 @@
+import _ from 'lodash';
+import React from 'react';
+import {sameWeek} from '../../dateutils';
+const commons = require('../commons');
+import XDate from 'xdate';
+
+const UPDATE_SOURCES = commons.UPDATE_SOURCES;
+// must be a positive number
+const NUMBER_OF_PAGES = 2;
+
+class Presenter {
```

Comment:
My concern here is that on my last PR I changed methods' implementations and removed others, and since this file is a new one it was not affected by the merge, which in turn can result in out-of-date methods.

Reply:
There are not many changes to WeekCalendar but it is worth double-checking: https://github.com/wix/react-native-calendars/pull/1490 I wrote comments for what I found, take a look to see if I covered it.